### PR TITLE
first implementation SearchString

### PR DESCRIPTION
**Pull request by amandanystedt**

### CODE REVIEW

File: SearchString.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    public static void main(String args[]){
+        String pattern;
+        String file;
+        if(args.length == 2){ 
+            pattern = args[0];
+            file = args[1];
+            try {
+                Scanner scan = new Scanner(new File(file));
+                while (scan.hasNextLine()) {
```
Comment (kevalane) [inline SearchString.java:16]: please comment what you are doing because I am slow in head

File: SearchString.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                    if (nextLine.contains(pattern)) {
+                        System.out.println(nextLine);
+                    }
+                }
+                scan.close(); // Close the scanner to release resources
+            } catch (FileNotFoundException e) {
+                System.out.println("File not found: " + file);
+            }
+        }else{
+            System.out.println("Wrong number of imputs");
```
Comment (kevalane) [inline SearchString.java]: Maybe tell us how to use, like "Usage: java SearchString <word> <file>"

File: SearchString.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+                    if (nextLine.contains(pattern)) {
+                        System.out.println(nextLine);
+                    }
+                }
+                scan.close(); // Close the scanner to release resources
+            } catch (FileNotFoundException e) {
+                System.out.println("File not found: " + file);
+            }
+        }else{
+            System.out.println("Wrong number of inputs");
```
Comment (kevalane) [inline SearchString.java]: New comment 2 on please add usage : ) "Usage: java SearchString <word> <file>"

File: SearchString.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        String pattern;
+        String file;
+        if(args.length == 2){ 
+            pattern = args[0];
+            file = args[1];
+            try {
+                Scanner scan = new Scanner(new File(file));
+                while (scan.hasNextLine()) {
+                    String nextLine = scan.nextLine();
+                    if (nextLine.contains(pattern)) {
```
Comment (kevalane) [inline SearchString.java:18]: can you make it case insensitive (using -i?)

**Review by kevalane — CHANGES_REQUESTED** [review]
Please fix above comments

File: SearchString.java (hunk 4 above)
Comment (amandanystedt) [reply SearchString.java:18]: No I don't think so, not today

**Review by kevalane — APPROVED** [review]
lgtm

**Pull request merged**